test/behavior/int_div.zig

```diff
@@ -0,0 +1,93 @@
+const std = @import("std");
+const builtin = @import("builtin");
+const expect = std.testing.expect;
+
+test "integer division" {
+    if (builtin.zig_backend == .stage2_aarch64) return error.SkipZigTest;
+    if (builtin.zig_backend == .stage2_c) return error.SkipZigTest;
+    if (builtin.zig_backend == .stage2_wasm) return error.SkipZigTest;
+    if (builtin.zig_backend == .stage2_arm) return error.SkipZigTest;
+
+    try testDivision();
+    comptime try testDivision();
+}
+fn testDivision() !void {
```

Comment:
potential edge cases: 
`0/1`,`-0/1`, `1/1`,`-1/-1`
`max/max`,`min/min`,`max/min`,`min/max`

if it is possible, add a TODO for `0/0`,`anynumber/0`

Reply:
Testing for ReleaseSafe is pointless at this stage (for native stage2 backends) as we don't have safety support of Sema implemented yet.

Reply:
Also, any UB checks will need to be postponed too until we get some safety guards emitted by Sema into codegen.